docs/cloud/regions.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: Available Regions
+sidebar_position: 4
+description: List of available regions in our Saleor Cloud offerings.
+---
+
+When creating your shop on Saleor Cloud, you can choose to host it in one of the following primary regions:
+
+| Primary Region (Country)    | Cloud Provider / Code             | Secondary (Failover) Region        |
+| --------------------------- | --------------------------------- | ---------------------------------- |
+| Europe - Ireland            | Amazon Web Services - `eu-west-1` | Europe - France (`eu-west-3`)      |
+| United States - N. Virginia | Amazon Web Services - `us-east-1` | United States - Ohio (`us-east-2`) |
```

Comment:
Although data in this table are accurate, I'm not a fun of how this table is formatted - in one row region ids are after `-`, in second one they are in `()`. WDYT about 👇🏿  
```suggestion
| Saleor region    | AWS primary region  | AWS secondary (failover) region |
| ---------------- | ------------------- | ------------------------------- |
| Ireland          | `eu-west-1`         | `eu-west-3`                     |
| N. Virginia, USA | `us-east-1`         | `us-east-2`                     |
```
(names of Saleor regions I took from our cloud console:

<img width="246" height="180" alt="Image" src="https://github.com/user-attachments/assets/12ba09bf-d576-4a5c-8571-ae817ffcadc4" />
)